js/main.js

```diff
@@ -25,37 +26,75 @@ var $newEntryForm = document.querySelector('#entry-form');
 var $entryList = document.querySelector('.entry-list');
 var $entry;
 
-function changeViewDOM() {
+function changeViewEntries() {
   data.view = 'entries';
   $dataViewEntries.className = 'view';
   $dataViewEntryForm.className = 'view hidden';
 }
 
+function changeViewForm() {
+  data.view = 'entry-form';
+  $dataViewEntries.className = 'view hidden';
+  $dataViewEntryForm.className = 'view';
+}
+
+var titleValue = $newEntryForm.elements.title.value;
+var notesValue = $newEntryForm.elements.notes.value;
+var imageValue = $newEntryForm.elements.url.value;
+var entryObj = {
+  url: imageValue,
+  title: titleValue,
+  notes: notesValue,
+  entryId: data.nextEntryId
+};
```

Comment:
This is in the global scope, I think it may not be supposed to be here.

Reply:
Moved to clickSave function.